### ISSUE TITLE
16.12 regression: `--keys` crashes with non-ascii arguments

### ISSUE THREAD

**Issue by robhagemans** [robhagemans/pcbasic#34]
```
../pcbasic.py --keys="£"
ERROR: Unhandled exception
Traceback (most recent call last):
  File "/home/rob/Projects/basic-project/pc-basic/pcbasic/main.py", line 33, in main
    run(*arguments)
  File "/home/rob/Projects/basic-project/pc-basic/pcbasic/main.py", line 69, in run
    launch_session(settings)
  File "/home/rob/Projects/basic-project/pc-basic/pcbasic/main.py", line 142, in launch_session
    kwargs=settings.get_launch_parameters())
  File "/home/rob/Projects/basic-project/pc-basic/pcbasic/config.py", line 559, in get_launch_parameters
    launch_params.update(self.get_session_parameters())
  File "/home/rob/Projects/basic-project/pc-basic/pcbasic/config.py", line 462, in get_session_parameters
    'keystring': self.get('keys').decode('string_escape').decode('utf-8'),
UnicodeEncodeError: 'ascii' codec can't encode character u'\xa3' in position 0: ordinal not in range(128)

```

**Comment by robhagemans** [comment]
16.12 only, works in 15.08

**Comment by robhagemans** [comment]
fixed by commit f8040f9
